sdk/inc/azure/core/az_http.h

```diff
@@ -182,12 +185,13 @@ AZ_NODISCARD AZ_INLINE az_result az_http_response_init(az_http_response* respons
  *
  * @see https://tools.ietf.org/html/rfc7230#section-3.1.2
  */
+// Member order is optimized for alignment.
 typedef struct
 {
+  az_span reason_phrase;
```

Comment:
Outside the scope of this PR, but these are public fields and need documentation.
https://azuresdkdocs.blob.core.windows.net/$web/c/docs/1.0.0-preview.3/structaz__http__response__status__line.html

Reply:
I created an issue to not lose this: https://github.com/Azure/azure-sdk-for-c/issues/994